src/overlays/misc/ovl_kaleido_scope/z_kaleido_debug.c

```diff
@@ -53,35 +180,35 @@ void KaleidoScope_DrawDebugEditorText(Gfx** gfxP) {
 
     GfxPrint_SetPos(&printer, 4, 2);
     GfxPrint_SetColor(&printer, 255, 60, 0, 255);
-    GfxPrint_Printf(&printer, "%s", GFXP_KATAKANA "ﾙﾋﾟｰ"); // "Rupee"
+    GfxPrint_Printf(&printer, "%s", T(GFXP_KATAKANA "ﾙﾋﾟｰ", "Rupee"));
```

Comment:
Im guessing the editor window is going to be very ineligible with T set to en, cause there is very little room for English characters. but oh well.